src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -74,10 +74,29 @@ typedef struct {
     /* 0x0C */ s16 yaw;
 } SpecialRespawnInfo; // size = 0x10
 
+typedef enum {
+    /*  1 */ ANIMSFX_TYPE_1 = 1,
+    /*  2 */ ANIMSFX_TYPE_2,
+    /*  3 */ ANIMSFX_TYPE_3,
+    /*  4 */ ANIMSFX_TYPE_4,
+    /*  5 */ ANIMSFX_TYPE_5,
+    /*  6 */ ANIMSFX_TYPE_6,
+    /*  7 */ ANIMSFX_TYPE_7,
+    /*  8 */ ANIMSFX_TYPE_8,
+    /*  9 */ ANIMSFX_TYPE_9
+} AnimSfxType;
```

Comment:
```suggestion
typedef enum {
    /* 1 */ ANIMSFX_TYPE_1 = 1,
    /* 2 */ ANIMSFX_TYPE_2,
    /* 3 */ ANIMSFX_TYPE_3,
    /* 4 */ ANIMSFX_TYPE_4,
    /* 5 */ ANIMSFX_TYPE_5,
    /* 6 */ ANIMSFX_TYPE_6,
    /* 7 */ ANIMSFX_TYPE_7,
    /* 8 */ ANIMSFX_TYPE_8,
    /* 9 */ ANIMSFX_TYPE_9
} AnimSfxType;
```
Extra space looks odd

Reply:
lol this is because of copy paste from mm, which added a 10th one. thanks